mmcv/ops/point_sample.py

```diff
@@ -153,8 +235,12 @@ def point_sample(input, points, align_corners=False, **kwargs):
     if points.dim() == 3:
         add_dim = True
         points = points.unsqueeze(2)
-    output = F.grid_sample(
-        input, denormalize(points), align_corners=align_corners, **kwargs)
+    if is_in_onnx_export_without_custom_ops():
```

Comment:
could add comment on why do this change here.

Reply:
Added.